esmvaltool/recipes/recipe_diurnal_temperature_index_wp7.yml

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# namelist_diurnal_temperature_index.yml
+#
+# Description
+# Namelist for computing the  diurnal temperature indicator.
+#
+# Authors
+# Alasdair Hunter (BSC, Spain)
+# Nicolau Manubens (BSC, Spain)
+# Louis-Philippe  (BSC, Spain)
+#
+# Project
+# C3S Magic
+#
+#Reference
+#http://secif.ipsl.fr/images/SECIF/documents/Communication/fiche_invulnerable/RC_indicateur_EN.pdf
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1960,  end_year: 1990}
```

Comment:
I cannot answer this question, sorry. The reason why I used it is that it was available in my local storage to test the recipe, but I don't have any confirmation about which data I should use/avoid.
Thanks for your comment, @bouweandela

Reply:
Practically I think you should use the data in /group_workspaces/jasmin2/cp4cds1/data/c3s-cmip5/output1/ on Jasmin, though I think at some point your input was asked about what data you intended to use so this could be made available in that directory. A spreadsheet was emailed around as well. Maybe some data from that directory or spreadsheet is missing though..